bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java

```diff
@@ -209,11 +209,9 @@ public void updateTransactions(Account account, Urllib urlopen) throws LoginExce
 			
 			account.setTransactions(transactions);
 		} catch (ClientProtocolException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
+			throw new BankException(e.getMessage(), e);
 		} catch (IOException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
-		}
+            throw new BankException(e.getMessage(), e);
+        }
```

Comment:
No, you probably have tabs set to 2 spaces, git have 4 spaces for tabs..

We should try to get rid of all tabs and use 2 spaces instead (seems to be default setting in *Android Studio*)